manager/orchestrator/restart/restart.go

```diff
@@ -190,6 +197,85 @@ func (r *Supervisor) Restart(ctx context.Context, tx store.Tx, cluster *api.Clus
 	return nil
 }
 
+// TaskRestartDelay calculates and returns the next delay based on t's RestartPolicy.
+// randomize is true if a backoff policy is being used,  false if the RestartPolicy
```

Comment:
The comment refers to `randomize`, but there doesn't seem to be a variable with that name.